recipesearch/src/Recipes/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+export class SearchResults extends Component {
+  render() {
+    const { searchResults } = this.props;
+    console.log(searchResults)
+    return (
+      <div>
+        {searchResults && searchResults.map((searchResult, index) => (
+          <div key={index}>
```

Comment:
Are you sure `index` is a reliable unique identifier?

Reply:
Let me work on using a better unique identifier

Reply:
Instead of index, I have put the mealId as the unique identifier